### ISSUE TITLE
Should work without superuser

### ISSUE THREAD

**Issue by bryanlarsen** [sastraxi/pgsh#75]
pgsh 0.10.6

Hosted SQL often comes without superuser access.   It'd be useful to use pgsh with a hosted DB.   `pgsh init` fails if a superuser is not supplied: it'd be nice if it succeeded and put pgsh into a limited functionality mode.

**Comment by sastraxi** [comment]
Can you expand a bit on your use case / the context for using a hosted database without superuser access during development? I want to make sure I understand the rationale behind this request so I can understand exactly what a limited functionality mode might look like.

**Comment by sastraxi** [comment]
@bryanlarsen are you able to run the following for your user? Working on a patch now that would check for the required permissions rather than explicitly for superuser.

```sql
GRANT EXECUTE ON FUNCTION pg_stat_file(text) TO ?;
ALTER ROLE ? CREATEDB;
```

**Comment by sastraxi** [comment]
I've closed the above and published [0.10.7](https://github.com/sastraxi/pgsh/releases/tag/v0.10.7) -- please let me know if this got you up-and-running.
